checkout-ui-custom/src/utils/services.js

```diff
@@ -71,7 +71,7 @@ export const getAddresses = async () => {
 };
 
 // GET Address by ID / Name?
-const getAddress = async (addressName, fields) => {
+export const getAddress = async (addressName, fields) => {
```

Comment:
we probably dont want to go down the master data level (and go straight to the index DB level)